index.html

```diff
@@ -877,6 +877,7 @@ <h2 id="BaseAudioContext">
     readonly        attribute double               currentTime;
     readonly        attribute AudioListener        listener;
     readonly        attribute AudioContextState    state;
+    readonly        attribute Worklet              worklet;
```

Comment:
Not sure if it's necessary, but it seems we're missing `[SameObject]` attribute.

Reply:
When does one use that? What does it mean?

Reply:
https://heycam.github.io/webidl/#SameObject

Once it's constructed/assigned by UA, the reference of `worklet` would not be changed. So `worklet` getter guarantees that you get the identical reference every time.